packages/shared/components/popups/Transaction.svelte

```diff
@@ -41,14 +36,30 @@
         return isFiat ? `${fiatAmount} (${iotaAmount})` : `${iotaAmount} (${fiatAmount})`
     }
 
+    $: accountOverview = $participationOverview?.find((apo) => apo?.accountIndex === $selectedAccount.index)
+    $: isAccountVoting = Object.values(accountOverview?.trackedParticipations)?.find((tp) =>
+        tp?.find((p) => p?.endMilestoneIndex === 0)
+    )
+
     let mustAcknowledgeGenericParticipationWarning
-    $: mustAcknowledgeGenericParticipationWarning = isAccountStaked(accountId) && isStakingPossible($stakingEventState)
+    $: mustAcknowledgeGenericParticipationWarning =
+        (isAccountStaked(accountId) && isStakingPossible($stakingEventState)) || isAccountVoting
+
+    let activeParticipationType
```

Comment:
To fully utilize TS we need to annotate the type of variables that are assigned values later.